Library/Formula/tachyon.rb

```diff
@@ -1,15 +1,15 @@
 class Tachyon < Formula
-  desc "Distributed storage system to enable data sharing across cluster frameworks"
+  desc "A memory-centric distributed storage system enabling reliable data sharing at memory-speed across cluster frameworks"
```

Comment:
Please shorten this description. “`tachyon: <desc>`” should fit under 80 characters.